pkgs/desktops/deepin/deepin-turbo/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, qtbase, deepin }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
```

Comment:
```suggestion
```
This isn't needed anymore.

Reply:
It is needed because of the following line:

```
passthru.updateScript = deepin.updateScript { inherit name; };
```

Reply:
See also https://github.com/NixOS/nixpkgs/pull/52403#issuecomment-448454551.

Reply:
Sorry about that.